dist/sequence-diagram.js

```diff
@@ -1498,7 +1498,7 @@ if (typeof Snap != 'undefined') {
     drawActor: function(actor, offsetY, height) {
       this.beginGroup();
       BaseTheme.prototype.drawActor.call(this, actor, offsetY, height);
-      return this.finishGroup().addClass('actor');
+      return this.finishGroup().addClass('actor actor'+actor.index);
```

Comment:
You should make this change in src/theme-snap.js#L231 not in dist/.  dist/ is generated from the files in src/, and can be regenerated by running `make`.

Otherwise LGTM.